src/libraries/System.Data.Odbc/src/System/Data/Odbc/OdbcCommand.cs

```diff
@@ -675,7 +675,6 @@ private OdbcDataReader ExecuteReaderObject(CommandBehavior behavior, string meth
                 bool mustRelease = false;
                 CNativeBuffer parameterBuffer = _cmdWrapper._nativeParameterBuffer;
 
-                RuntimeHelpers.PrepareConstrainedRegions();
```

Comment:
This project targets netstandard + netframework.
(This comment applies file-wide.)

Reply:
The file is in an ItemGroup that only targets NetCoreApp.